### PR TITLE
Add Retroactive Protocol Revenue Calculation Script

### PR DESCRIPTION
**Pull request by chuddster**

### CODE REVIEW

File: scripts/calculate-retroactive-protocol-rewards.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+let AMOUNT_FOR_GAMMA_UNIV3_SEED = BigNumber.from("180").mul(
+  BigNumber.from(10).pow(18)
+);
+
+async function main() {
+  let tokens = [
+    "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0", // wsteth
+    "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599", // wbtc
+    "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", // usdc
+    "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9", // aave
```
Comment (kenchangh) [inline scripts/calculate-retroactive-protocol-rewards.ts:19]: Need to add yvUSDC into the list of tokens

File: scripts/calculate-retroactive-protocol-rewards.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  ];
+  let balancesBefore = [];
+  let balancesAfter = [];
+
+  for (var token of tokens) {
```
Comment (kenchangh) [inline scripts/calculate-retroactive-protocol-rewards.ts:24]: This is missing the ETH balance for the multisig

File: scripts/calculate-retroactive-protocol-rewards.ts (hunk 1 above)
Comment (chuddster) [reply scripts/calculate-retroactive-protocol-rewards.ts:19]: added

File: scripts/calculate-retroactive-protocol-rewards.ts (hunk 2 above)
Comment (chuddster) [reply scripts/calculate-retroactive-protocol-rewards.ts:24]: make that call right after for loop

**Review by kenchangh — APPROVED** [review]

**Pull request merged**